### PR TITLE
Dev dlangst

### PR DESCRIPTION
**Pull request by dlangst**

### CODE REVIEW

**Review by dcwebermn — APPROVED** [review]
LGTM

File: src/pages/installing/installing.md [hunk 1]
```diff
@@ -64,14 +58,14 @@ To add the pull secret to the Openshift global pull secret:
 
 1. Create a base64 encoded string with the registry userid and password as it aligns with your access method.
```
Comment (cagrev) [inline src/pages/installing/installing.md:59]: Remove everything in this section from here on. This is specific to the IBM registry, not Azure.

File: src/pages/installing/license-tracking.md [hunk 2]
```diff
@@ -5,31 +5,28 @@ categories: installing
...
 
 ## Deploying License Service
 
 If [Validating if License Service is deployed on the cluster](#validating-if-license-service-is-deployed-on-the-cluster) determines that the License Service is not deployed on the cluster where you have deployed ACD,
 or the License Service does not return a status of Running, refer to the information about License Service, including how to install, retrieve license usage data, and troubleshoot.
-See the [License Service](https://github.com/IBM/ibm-licensing-operator/blob/master/docs/License_Service_main.md) documentation. If deploying the License Service to a disconnected or air-gapped cluster, see the [offline installation License Service](https://github.com/IBM/ibm-licensing-operator/blob/latest/docs/Content/Install_offline.md) documentation.
+See the [License Service](https://github.com/IBM/ibm-licensing-operator/blob/master/docs/License_Service_main.md) documentatThe IBM ion. If deploying the License Service to a disconnected or air-gapped cluster, see the [offline installation License Service](https://github.com/IBM/ibm-licensing-operator/blob/latest/docs/Content/Install_offline..
```
Comment (cagrev) [inline src/pages/installing/license-tracking.md:25]: Copy/paste error. "documentatThe IBM ion"

File: src/pages/installing/uninstalling-ibm.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+---
+title: "Uninstalling ACD"
+excerpt: "Uninstalling ACD."
+categories: installing
+slug: uninstalling
+toc: true
+---
+_Note: The following instructions, including the cloudctl command line utility, are for uninstalling IBM Annotator for Clinical Data Container Edition._
```
Comment (cagrev) [inline src/pages/installing/uninstalling-ibm.md:8]: The web console path should be in the uninstall ACD section and move only the cloudctl/cli uninstall to the IBM-specific page. You can also reference the web console uninstall path from the IBM uninstall page.

File: src/pages/installing/uninstalling.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ slug: uninstalling
 toc: true
 ---
 
+_Note: Refer here for uninstalling instructions for [IBM Annotator for Clinical Data Container Edition](/installing/uninstalling-ibm/)._
```
Comment (cagrev) [inline src/pages/installing/uninstalling.md:9]: Move this down, after the web console steps to uninstall. Then say something like "To uinstall an ACD instance that was installed with IBM's cloudctl CLI, refer here ..."

**Review by cagrev — APPROVED** [review]
A few minor comments. Please make those changes and deliver. Thanks.

**Pull request merged**